tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -367,35 +367,33 @@ constexpr bool test() {
-        static_assert(!is_constructible_v<span<int, 3>, C, Z>);
-        static_assert(!is_constructible_v<span<int, 3>, Z, I>);
-        static_assert(!is_constructible_v<span<int, 3>, Z, C>);
-        static_assert(!is_constructible_v<span<int, 3>, Z, Z>);
-
-        static_assert(is_constructible_v<span<const int>, I, I>);
-        static_assert(is_constructible_v<span<const int>, I, C>);
-        static_assert(is_constructible_v<span<const int>, C, I>);
-        static_assert(is_constructible_v<span<const int>, C, C>);
-
-        static_assert(!is_constructible_v<span<int>, I, int*>);
-        static_assert(!is_constructible_v<span<int>, int*, I>);
+        static_assert(is_nothrow_constructible_v<span<int>, I, I>); // strengthened 
+        static_assert(is_nothrow_constructible_v<span<int>, I, C>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int>, I, Z>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int>, C, I>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int>, C, C>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int>, C, Z>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int>, Z, I>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int>, Z, C>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int>, Z, Z>);
+
+        static_assert(is_nothrow_constructible_v<span<int, 3>, I, C>); // strengthened 
+        static_assert(is_nothrow_constructible_v<span<int, 3>, I, I>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int, 3>, I, Z>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int, 3>, C, I>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int, 3>, C, C>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int, 3>, C, Z>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int, 3>, Z, I>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int, 3>, Z, C>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int, 3>, Z, Z>); // strengthened 
+
+        static_assert(is_nothrow_constructible_v<span<const int>, I, I>); // strengthened 
+        static_assert(is_nothrow_constructible_v<span<const int>, I, C>); // strengthened 
+        static_assert(is_nothrow_constructible_v<span<const int>, C, I>); // strengthened 
+        static_assert(is_nothrow_constructible_v<span<const int>, C, C>); // strengthened 
+
+        static_assert(!is_nothrow_constructible_v<span<int>, I, int*>); // strengthened 
+        static_assert(!is_nothrow_constructible_v<span<int>, int*, I>); // strengthened 
 
         static_assert(is_nothrow_constructible_v<span<Base>, Base*, Base*>); // strengthened
         static_assert(is_nothrow_constructible_v<span<Base, 3>, Base*, Base*>); // strengthened

```

Comment:
There are two more positive cases on lines 405-406 that should be strengthened to `is_nothrow_constructible_v`.